pkg/security/probe/model.go

```diff
@@ -701,7 +757,7 @@ func (e *ExecEvent) ResolvePPID(event *Event) int {
 
 // ResolveInode resolves the inode to a full path
 func (e *ExecEvent) ResolveInode(event *Event) string {
-	if len(e.PathnameStr) == 0 {
+	if len(e.PathnameStr) == 0 && event != nil {
```

Comment:
As discussed, event might be nil in some `Resolve*` calls, so this is just to be safe, and so that we know we can safely call all the `Resolve*` with a nil event.